Source/WebGPU/WebGPU/Device.h

```diff
@@ -81,6 +81,17 @@ using GPUShaderValidation = MTLShaderValidation;
 using GPUShaderValidation = uint32_t;
 #endif
 
+#define WEBKIT_DRAW_INDIRECT_STRUCT_TYPES \
```

Comment:
the define in a header file seems preferable to potentially conflicting definitions in C++ and MSL